Client/Frontend/Library/LibraryViewController/LibraryViewController.swift

```diff
@@ -135,6 +131,12 @@ class LibraryViewController: UIViewController, Themeable {
         LegacyThemeManager.instance.statusBarStyle
     }
 
+    func resetHistoryPanelPagination() {
+        if CoordinatorFlagManager.isLibraryCoordinatorEnabled {
```

Comment:
The old behaviour was to reset reset history panel pagination every time we presented so I moved it to it's separate method.